tests/core/gadget-kernel-refs-update-pc/task.yaml

```diff
@@ -22,6 +22,7 @@ environment:
     START_REVISION: 1000
     # uploading a large snap makes OOM kill snapd
     SNAPD_NO_MEMORY_LIMIT: 1
+    CONF_OVERRIDE: /etc/systemd/system/snapd.service.d/90-$(basename $SPREAD_TASK).conf
```

Comment:
perhaps we could use "current" or something like that instead of "$(basename $SPREAD_TASK)"
So then it is easier to find it in case we need to add some generic restore/debug

/etc/systemd/system/snapd.service.d/90-current.conf

Reply:
should the files leak for any reason we'll know which task left them, otherwise it'd be `current` for any task including the one executing